ast/meson.build

```diff
@@ -13,8 +13,9 @@ ast_dep = declare_dependency(include_directories : '.',
 
 libvisitors = static_library('visitors', 
                              sources : [
-                                 'impl/print_visitor.cpp',
-                                 'impl/identifier_resolver.cpp'
+                                 'impl/ast_printer.cpp',
```

Comment:
You forgot the `print_visitor` (which actually I don't know what it does exactly now that we have both `ast_printer` and `pretty_printer`)

Reply:
I renamed `PrintVisitor` to 'AstPrinter`. So the `ast_printer.cpp` *is* the `pretty_printer.cpp`.